### ISSUE TITLE
Unable to connect to CRM

### ISSUE THREAD

**Issue by dirasmitabasu** [MscrmTools/MscrmTools.Xrm.Connection#163]
Unable to connect to CRM: Method not found: 'System.String Microsoft.Xrm.Sdk.Organization.OrganizationDetail.get_SchemaType()'.

Method not found: 'System.String Microsoft.Xrm.Sdk.Organization.OrganizationDetail.get_SchemaType()'.Unable to Login to Dynamics CRM

The above error comes up when trying to connect to Dataverse. Doesn't matter what type of connection I try, Microsoft Login Control or Client ID secret or OAuth/MFA, all have the same behavior. These were all working before the update.
![image](https://github.com/user-attachments/assets/89da95b3-e0bc-4103-b6f9-56335dfb5f02)

**Comment by MscrmTools** [comment]
Can you give me the version of connection controls you are using?

**Comment by dirasmitabasu** [comment]
I installed the latest version of the toolbox and then this started
happening. Where can I get the connection controls version?

**Comment by MscrmTools** [comment]
In Settings

![image](https://github.com/user-attachments/assets/13f61535-76e5-4f8d-9e82-585b7e6f6f21)

**Comment by dirasmitabasu** [comment]
![image](https://github.com/user-attachments/assets/1ee96c50-fb8f-472b-9179-a9fb9fb2fa5f)

Assembly									Version
XrmToolBox								1.2023.12.68
XrmToolBox.Extensibility					        1.2023.12.68
XrmToolBox.ToolLibrary						1.2023.12.68
McTools.Xrm.Connection					1.2024.7.58
McTools.Xrm.Connection.WinForms			1.2024.7.58
Microsoft.Web.WebView2.WinForms			1.0.1518.46
Microsoft.Xrm.Sdk							9.2.49.3165
Microsoft.Xrm.Tooling.Connector				4.0.0.188
Newtonsoft.Json							13.0.2.27524
WeifenLuo.WinFormsUI.Docking				3.0.6.0
WeifenLuo.WinFormsUl.Docking.ThemeVS2015	3.0.6.0

**Comment by MscrmTools** [comment]
OK, so these McTools.Xrm.Connection assemblies are beta one and should be removed...

Can you check your settings for the category Connection Controls and see if you are using pre release?

**Comment by dirasmitabasu** [comment]
Yes, I was enrolled in Pre-release. I un checked it and restarted the application, I am still getting the error. 
I downloaded the previous release, and that one works.
